### DISCUSSION TITLE
Cash Flow Graph

### DISCUSSION THREAD

**Discussion by Mozenraath** [ananthakumaran/paisa#144] [General]
From today evening the cash flow graph suddenly changed the scale (Y Axis). both + & - ve values on Y axis.

![image](https://github.com/ananthakumaran/paisa/assets/155997609/039d8f7b-4885-4982-98bb-37f37e1a8e76)

**Comment by ananthakumaran** [comment]
Strange, you are paying off your liabilities? (loan or something?). I will try to check, in the mean time, if you can reproduce the issue with a small ledger, let me know.

**Reply by ananthakumaran** [reply]
@Mozenraath Paying off liabilities was not handled properly when computing the Y axis, I have pushed a fix to master https://github.com/ananthakumaran/paisa/commit/680b500d1a51a99bba9e302200167816b321facf

**Reply by Mozenraath** [reply]
Yes..  I was returning money that was borrowed. And I kept it under liability so that it wont dominate my expenses charts.

**Reply by Mozenraath** [reply]
I still see the graph as posted above. no changes so far.

**Reply by ananthakumaran** [reply]
Are you building the app from the master branch? Note that the changes have not yet been released, and will get included in the next release.

**Reply by Mozenraath** [reply]
No. Launching from Pikapods... So would have to wait for the release.
